Context.sublime-menu

```diff
@@ -27,6 +27,10 @@
 				"caption": "Convert inline link to reference",
 				"command": "mde_convert_inline_link_to_reference"
 			},
+			{
+				"caption": "Convert bare link to markdown",
+				"command": "mde_convert_bare_link_to_md_link"
```

Comment:
Those links are called "autolink" (see: https://spec.commonmark.org/0.30/#autolinks)

Reply:
Autolinks are links surrounded by brackets, in the form of `<http://foo.bar.baz>`. What I've written here specifically targets bare urls, as in https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

Reply:
My source of spec is https://github.github.com/gfm/#autolinks-extension-

Reply:
That link describes the "autolinks" markdown extension extension that GFM uses, which isn't "the spec". We can still call these autolinks if you want, but I think that may be confusing for anyone who wasn't involved in this conversation.

Reply:
It's the spec of GFM. Furthermore as end user I'd expect this function to work for both, CM and GFM autolinks.

- `https://server.com/site` -> `[link title](https://server.com/site)`
- `<https://server.com/site>` -> `[link title](<https://server.com/site>)`

Hence using `mde_convert_autolink_to_md_link` for both wouldn't be too bad.